tools/sof-kernel-log-check.sh

```diff
@@ -355,6 +355,12 @@ ignore_str="$ignore_str"'|elan_i2c i2c-ELAN0000:.*: invalid report id data'
 # iwlwifi 0000:00:14.3: No beacon heard and the time event is over already...
 ignore_str="$ignore_str"'|iwlwifi [[:digit:].:]+: '
 
+# i915 AUX logs can be ignored
+# origin logs seen on ADLS platforms
+# i915 0000:00:02.0: [drm] *ERROR* AUX A/DDI A/PHY A: did not complete or timeout within 10ms (status 0xad4003ff)
+# i915 0000:00:02.0: [drm] *ERROR* AUX A/DDI A/PHY A: not done (status 0xad4003ff)
+ignore_str="$ignore_str"'|i915 [[:digit:].:]+: \[drm\] \*ERROR\* AUX .+'
+
```

Comment:
This is a bit wide, can you limit this to adl platform only (under "case "$platform"")..?

Reply:
> This is a bit wide, can you limit this to adl platform only (under "case "$platform"")..?

yes I will refine it